4337/contracts/test/SafeMock.sol

```diff
@@ -307,7 +307,7 @@ contract Safe4337Mock is SafeMock {
         checkSignatures(operationHash, operationData, userOp.signature);
     }
 
-    function validateReplayProtection(UserOperation calldata userOp) internal {
+    function validateReplayProtection(UserOperation calldata userOp) internal view {
```

Comment:
Fixes a Solidity compiler warning that the access of the function could be restricted.